src/components/title.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react'
+import styled from 'styled-components'
+import * as fonts from '../assets/fonts'
+
+type TitleSize = "small" | "medium" | "large" | "xlarge" | "xxlarge"
+
+interface TitleProps extends React.HTMLProps<JSX.Element> {
+  titleSize?: TitleSize
+}
+
+const titleSizes = {
+  small: '25px',
+  medium: '30px',
+  large: '37px',
+  xlarge: '50px',
+  xxlarge: '72px'
+}
+
+const Title: React.SFC<TitleProps> = props => (
+  <div className={props.className}>
```

Comment:
I don't actually have strong opinions on `h1`-`h6` vs `div`. The `p` tag on the other hand may have some default CSS properties and would require us to add extra overrides. So as long as we use `h*` or `div`, I'm 100% 👍.